src/index.ts

```diff
@@ -223,7 +248,7 @@ function calculateSortedComponentsFromOutput() {
     components.sort((a, b) => b.magnitude - a.magnitude);
 }
 
-function redraw() {
+function redraw(implexity: number = complexity, intrameter: number = parameter) {
```

Comment:
I'm not familiar with what's going on here. 'Implexity' and 'Intrameter'—do they have English meanings or is there another way we could achieve the same goal?

Reply:
Yep, the naming is lame. That was a portmanteau between `internal`+complexity/parameter, only as a proof of concept that indeed requires some more attention.

Reply:
b2a5a5a - underscored global complexity and parameter names